scss/ionic.app.scss

```diff
@@ -21,3 +21,6 @@ $ionicons-font-path: "../lib/ionic/fonts" !default;
 // Include all of Ionic
 @import "www/lib/ionic/scss/ionic";
 
+// Application Styles
+@import "www/scss/rcredits";
```

Comment:
Moved, but still not fixing live update issue. Mealyzer mobile is using css, not scss it looks like for custom styles.